manifests/prometheus/spec/operations/monitor_cf_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe "Monitor CF" do
+  it "adds the firehose instance_group" do
+    firehose = manifest_with_defaults.get("instance_groups.prometheus2")
+    expect(firehose).not_to be_nil
+    expect(firehose["vm_type"]).to eq("medium")
+    expect(firehose["networks"]).to eq([{"name" => "cf"}])
```

Comment:
```suggestion
    expect(firehose["networks"]).to eq([{ "name" => "cf" }])
```
☝️ to make the linter happy

Reply:
Yeah, rubocop... sigh... It goes too far sometime IMO.

I've rebased this change in...